person.rb

```diff
@@ -0,0 +1,25 @@
+# Person class
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
```

Comment:
Test out if you need the `false` here and below. 

By default. I think `nil` is returned at the end of the function, which is falsy.

Reply:
Yeah, i read its better practice to use false when using booleans not passing to something else

Reply:
Makes sense 👍